workitem/link/link_repository.go

```diff
@@ -91,7 +93,7 @@ func (r *GormWorkItemLinkRepository) ValidateCorrectSourceAndTargetType(ctx cont
 
 // Create creates a new work item link in the repository.
 // Returns BadParameterError, ConversionError or InternalError
-func (r *GormWorkItemLinkRepository) Create(ctx context.Context, sourceID, targetID uint64, linkTypeID satoriuuid.UUID) (*app.WorkItemLinkSingle, error) {
+func (r *GormWorkItemLinkRepository) Create(ctx context.Context, sourceID, targetID uint64, linkTypeID satoriuuid.UUID, creatorID satoriuuid.UUID) (*app.WorkItemLinkSingle, error) {
```

Comment:
I initially added a `creatorID`/`modifierID`/`suppressorID` parameter to the repository methods to avoid having to retrieve the current user from the context via the `login.ContextIdentity(context.Context)` function, as @aslakknutsen requested (the functions  in the `login` having nothing to do ar the repository level). Also, it makes this parameter more visible in the method signature. But I also like your idea of embedding it in the `Context`.